shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Serialization.hs

```diff
@@ -1,293 +1,6 @@
-{-# LANGUAGE BangPatterns #-}
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE FlexibleInstances #-}
-{-# LANGUAGE GeneralizedNewtypeDeriving #-}
-{-# LANGUAGE LambdaCase #-}
-{-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE Rank2Types #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-
 module Shelley.Spec.Ledger.Serialization
```

Comment:
If we're moving this module, we shouldn't move it to `cardano-ledger-core`, but to https://github.com/input-output-hk/cardano-base/blob/master/binary/src/Cardano/Binary/ instead. Pretty much none of this is ledger specific. We should also review to see how much of it is actually used.